doc/.rtd-environment.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
     - scipy
     - "astropy>=3.1"
     - graphviz
+    - pytables
```

Comment:
This .yml file defines what is needed in order to generate the readthedocs documentation (HTML and PDF files).  The 'setup.py' file also needs to be updated in order to insure that the right packages get installed when Drizzlepac gets installed next.

Reply:
Resolved in commit [af26058](https://github.com/spacetelescope/drizzlepac/pull/682/commits/af26058f4c31b40e1f1792634717a14ed8040aed)

Reply:
resolved in commit [e880568](https://github.com/spacetelescope/drizzlepac/pull/682/commits/e880568a9f6d815baf7581425bd892e56cdf55f9)